src/main/java/com/rockwell/basics/animals/Application.java

```diff
@@ -0,0 +1,19 @@
+package com.rockwell.basics.animals;
+
+import com.rockwell.basics.animals.bird.Hoopoe;
+import com.rockwell.basics.animals.mammal.Lion;
+import com.rockwell.basics.animals.mammal.Tiger;
+
+public class Application {
+    public static void main(String[] args) {
```

Comment:
Instead of this let's think about addig some tests. We are unsure about where this implementation will go in the future. We are sure about Animal (and extending classes) API. This is something we can start to test as it is a stightforward and easy thing to do.

Reply:
Do You mean that this class should be deleted?